### DISCUSSION TITLE
Add dialog if the target folder is undefined

### DISCUSSION THREAD

**Discussion by RafaelLinux** [setvisible/DownZemAll#214] [Q&A]
Recently, I discovered that DZA (including 3.0.2) doesn't check manual entered path to download folder. If folders doesn't exist, DZA creates a new folder in "Documents" user folders, without asking about.

Please, do check folder exist and doesn't create it without asking user if it is what he wants.

Thank you

**Comment by setvisible** [comment]
Hi

It's normal because the mask is necessary to define the destination. Without mask the default folder is the Documents.

It's more something to do with the users configuration of the mask.

Personally I don't want a popup every time ArrowDL creates a new directory. I generally organize my mask with the *subdirs*/*name*.*ext* in order to preserve the website organization, and also avoid overwriting when 2 files have the same name in different folders.

I'll move this topic to "Discussions".

Regards
